tests/FakeItEasy.Specs/ConfigurationSpecs.cs

```diff
@@ -697,6 +697,7 @@ public virtual void DoSomething()
 
             public void DoSomethingNonVirtual()
             {
+                this.WasCalled = true;

```

Comment:
Thanks! Now I get it.

It sounds like it would be better to add a suppression for this method.